src/main/java/uk/gov/hmcts/reform/finrem/notifications/controllers/NotificationController.java

```diff
@@ -128,4 +129,20 @@
         emailService.sendConfirmationEmail(notificationRequest, FR_CONTESTED_HWF_SUCCESSFUL);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+
+    @PostMapping(path = "/contested/application-issued", consumes = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "send e-mail for Application Issued.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "Application Issued e-mail sent successfully")})
+    public ResponseEntity<Void> sendContestedEmailApplicationIssued(
+            @RequestBody
+            @ApiParam(value = "The fixtures contains case reference number,"
+                    + " solicitorReferenceNumber and the email address that will receive "
+                    + "the notification that the application has been issued and all are mandatory")
+            final NotificationRequest notificationRequest) {
+        log.info("Received request for notification email for ApplicationIssued. Notification request : {}",
```

Comment:
Can we log the journey type in here also, Consented and Contested now have notifications for this. Would be useful to see this reflected in logs